packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -4,6 +4,72 @@ import KeyEncoder from 'key-encoder'
 import { compressedPubKey } from '@celo/utils/lib/dataEncryptionKey'
 import { hexToBuffer, trimLeading0x } from '@celo/utils/lib/address'
 
+interface DeleteFinclusiveBankAccountParams {
+  accountMTWAddress: string
+  dekPrivate: string
+  id: number
+}
+
+/**
+ * get a fiat bank account from finclusive
+ *
+ *
+ * @param {params.accountMTWAddress} accountAddress
+ * @param {params.dekPrivate} dekPrivate private data encryption key
+ * @returns {Response} response object from the fetch call
+ */
+export const deleteFinclusiveBankAccount = async ({
+  accountMTWAddress,
+  dekPrivate,
+  id,
+}: DeleteFinclusiveBankAccountParams) => {
+  const body = {
+    accountAddress: accountMTWAddress,
+    accountId: id,
+  }
+  return signAndFetch({
+    path: `/account/bank-account?accountAddress=${encodeURIComponent(accountMTWAddress)}`,
+    accountMTWAddress,
+    dekPrivate,
+    requestOptions: {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(body),
+    },
+  })
+}
+
+interface GetFinclusiveBankAccountParams {
+  accountMTWAddress: string
+  dekPrivate: string
+}
+
+/**
+ * get a fiat bank account from finclusive
+ *
+ *
+ * @param {params.accountMTWAddress} accountAddress
+ * @param {params.dekPrivate} dekPrivate private data encryption key
+ * @returns {Response} response object from the fetch call
+ */
+export const getFinclusiveBankAccount = async ({
```

Comment:
since this may include info about multiple accounts, I suggest a plural "accounts" in the name.

Also, I wonder if the return type should be more explicit than just a "Response" that the caller needs to know all the fields on. Would be nice if this module did a bit more encapsulation of those IHL-specific details.

I realize the other functions in the module don't do this already. No need to refactor everything in this PR. If you agree with the suggestion, then you could refactor just this function and punt on the others, or refactor all of the functions returning a "response" type (including this one) in a separate PR.

Reply:
Good call on the (s) for this function. I'll do a quick followup to add a detailed return object on all of them